docs/fastdds/getting_started/simple_app/includes/task_publisher.rst

```diff
@@ -0,0 +1,157 @@
+Write the Fast DDS publisher
+""""""""""""""""""""""""""""
+
+From the `src` directory in the workspace, run the following command to download the HelloWorldPublisher.cpp file.
+
+.. code-block:: bash
+
+    wget -O HelloWorldPublisher.cpp \
+        https://raw.githubusercontent.com/eProsima/Fast-RTPS-docs/master/code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+
+Now you have the publisher's source code. The publisher is going to send 10 publications under the topic HelloWorld.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+  :language: C++
+  :linenos:
+
+Examine the code
+****************
+
+At the beginning of the file we have a Doxygen style comment block with the ``@file`` field that tells us the name of
+the file.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 15-18
+
+Below are the includes of the C++ headers.
+The first one includes the HelloWorldPubSubTypes.h file with the serialization and deserialization functions of the
+data type that we have defined in the previous section.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 20
+
+The next block includes the C++ header files that allow the use of the Fast DDS API.
+
+*   DomainParticipantFactory. Allow the creation and destruction of DomainParticipant objects.
+*   DomainParticipant. Acts as a container for all other Entity objects and as a factory for the Publisher, Subscriber,
+    and Topic objects.
+*   TypeSupport. Provide the participant with the methods to serialize, deserialize and get the key of a
+    specific data type.
+*   Publisher. Is the object responsible for the actual dissemination of publications.
+*   DataWriter. Allows the application to set the value of the data to be published under a given Topic.
+*   DataWriterListener. It allows the redefinition of the functions of the DataWriterListener.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 22-27
+
+Next, we define the namespace that contains the eProsima Fast DDS classes and functions that we are going to use in
+our application.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 29
+
+The next line creates the HelloWorldPublisher class that implements a publisher.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 31
+
+The public constructor and destructor of the class is defined below. The fields of the class will be the participant,
+the publisher, the topic, the data writer, and the data type.
+The constructor initializes the private fields of the class to ``nullptr`` except the TypeSupport object, that
+initializes an object of the HelloWorldPubSubType class.
```

Comment:
Every time you name a class, you can do so with :class:`HelloWorldPubSubType`. This way we create a more uniform view of the documentation